spec/system/signup/signup_spec.rb

```diff
@@ -9,10 +9,11 @@
 
     test_email = gmail.account_email.gsub(/@/, "+#{Time.now.to_i}@")
 
-    if ENV["STAGING"] 
+    if (ENV["STAGING"] = 1)
```

Comment:
Also, no need for `()` around condition - `if ENV["STAGING"]` is sufficient - another one rubocop should yell about.

Reply:
done